### PR TITLE
Improve error message in `test_schema_is_up_to_date`

### PR DESCRIPTION
**Pull request by harupy**
Signed-off-by: harupy <hkawamura0130@gmail.com>

## Related Issues/PRs

#xxx

## What changes are proposed in this pull request?

Improve error message in `test_schema_is_up_to_date` to make it easier to update DB schema files without using Docker.

## How is this patch tested?

- [x] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change the documentation?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure the changed pages / sections render correctly by following the steps below.

1. Click the `Details` link on the `Preview docs` check.
2. Find the changed pages / sections and make sure they render correctly.

### Is this a user-facing change?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/pipelines`: Pipelines, Pipeline APIs, Pipeline configs, Pipeline Templates
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface
- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: tests/db/test_schema.py [hunk 1]
```diff
@@ -136,6 +137,23 @@ def test_schema_is_up_to_date():
...
+{latest_schema}
+==================================== ACTUAL ====================================
+{existing_schema}
+===================================== DIFF =====================================
+{diff}
+================================== HOW TO FIX ==================================
+Manually copy & paste the expected schema in {rel_path} or run the following command:
+{update_command}
+"""
     assert schema_equal(existing_schema, latest_schema), message
```
Comment (harupy) [inline tests/db/test_schema.py:157]:
```
E       AssertionError: 
E         =================================== EXPECTED ===================================
E         
E         CREATE TABLE alembic_version (
E               version_num VARCHAR(32) NOT NULL,
E               CONSTRAINT alembic_version_pkc PRIMARY KEY (version_num)
E         )
E         
E         
E         CREATE TABLE experiments (
E               experiment_id INTEGER NOT NULL,
E               name VARCHAR(256) NOT NULL,
E               artifact_location VARCHAR(256),
E               lifecycle_stage VARCHAR(32),
E               creation_time BIGINT,
E               last_update_time BIGINT,
E               CONSTRAINT experiment_pk PRIMARY KEY (experiment_id),
E               UNIQUE (name),
E               CONSTRAINT experiments_lifecycle_stage CHECK (lifecycle_stage IN ('active', 'deleted'))
E         )
E         
E         
E         CREATE TABLE registered_models (
E               name VARCHAR(256) NOT NULL,
E               creation_time BIGINT,
E               last_updated_time BIGINT,
E               description VARCHAR(5000),
E               CONSTRAINT registered_model_pk PRIMARY KEY (name),
E               UNIQUE (name)
E         )
E         
E         
E         CREATE TABLE experiment_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               experiment_id INTEGER NOT NULL,
E               CONSTRAINT experiment_tag_pk PRIMARY KEY (key, experiment_id),
E               FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id)
E         )
E         
E         
E         CREATE TABLE model_versions (
E               name VARCHAR(256) NOT NULL,
E               version INTEGER NOT NULL,
E               creation_time BIGINT,
E               last_updated_time BIGINT,
E               description VARCHAR(5000),
E               user_id VARCHAR(256),
E               current_stage VARCHAR(20),
E               source VARCHAR(500),
E               run_id VARCHAR(32),
E               status VARCHAR(20),
E               status_message VARCHAR(500),
E               run_link VARCHAR(500),
E               CONSTRAINT model_version_pk PRIMARY KEY (name, version),
E               FOREIGN KEY(name) REFERENCES registered_models (name) ON UPDATE CASCADE
E         )
E         
E         
E         CREATE TABLE registered_model_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               name VARCHAR(256) NOT NULL,
E               CONSTRAINT registered_model_tag_pk PRIMARY KEY (key, name),
E               FOREIGN KEY(name) REFERENCES registered_models (name) ON UPDATE CASCADE
E         )
E         
E         
E         CREATE TABLE runs (
E               run_uuid VARCHAR(32) NOT NULL,
E               name VARCHAR(250),
E               source_type VARCHAR(20),
E               source_name VARCHAR(500),
E               entry_point_name VARCHAR(50),
E               user_id VARCHAR(256),
E               status VARCHAR(9),
E               start_time BIGINT,
E               end_time BIGINT,
E               source_version VARCHAR(50),
E               lifecycle_stage VARCHAR(20),
E               artifact_uri VARCHAR(200),
E               experiment_id INTEGER,
E               deleted_time BIGINT,
E               CONSTRAINT run_pk PRIMARY KEY (run_uuid),
E               FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id),
E               CONSTRAINT runs_lifecycle_stage CHECK (lifecycle_stage IN ('active', 'deleted')),
E               CONSTRAINT source_type CHECK (source_type IN ('NOTEBOOK', 'JOB', 'LOCAL', 'UNKNOWN', 'PROJECT')),
E               CHECK (status IN ('SCHEDULED', 'FAILED', 'FINISHED', 'RUNNING', 'KILLED'))
E         )
E         
E         
E         CREATE TABLE latest_metrics (
E               key VARCHAR(250) NOT NULL,
E               value FLOAT NOT NULL,
E               timestamp BIGINT,
E               step BIGINT NOT NULL,
E               is_nan BOOLEAN NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT latest_metric_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid),
E               CHECK (is_nan IN (0, 1))
E         )
E         
E         
E         CREATE TABLE metrics (
E               key VARCHAR(250) NOT NULL,
E               value FLOAT NOT NULL,
E               timestamp BIGINT NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               step BIGINT DEFAULT '0' NOT NULL,
E               is_nan BOOLEAN DEFAULT '0' NOT NULL,
E               CONSTRAINT metric_pk PRIMARY KEY (key, timestamp, step, run_uuid, value, is_nan),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid),
E               CHECK (is_nan IN (0, 1))
E         )
E         
E         
E         CREATE TABLE model_version_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               name VARCHAR(256) NOT NULL,
E               version INTEGER NOT NULL,
E               CONSTRAINT model_version_tag_pk PRIMARY KEY (key, name, version),
E               FOREIGN KEY(name, version) REFERENCES model_versions (name, version) ON UPDATE CASCADE
E         )
E         
E         
E         CREATE TABLE params (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(500) NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT param_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid)
E         )
E         
E         
E         CREATE TABLE tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT tag_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid)
E         )
E         
E         ==================================== ACTUAL ====================================
E         
E         CREATE TABLE alembic_version (
E               version_num VARCHAR(32) NOT NULL,
E               CONSTRAINT alembic_version_pkc PRIMARY KEY (version_num)
E         )
E         
E         
E         CREATE TABLE experiments (
E               experiment_id INTEGER NOT NULL,
E               name VARCHAR(256) NOT NULL,
E               artifact_location VARCHAR(256),
E               lifecycle_stage VARCHAR(32),
E               last_update_time BIGINT,
E               CONSTRAINT experiment_pk PRIMARY KEY (experiment_id),
E               UNIQUE (name),
E               CONSTRAINT experiments_lifecycle_stage CHECK (lifecycle_stage IN ('active', 'deleted'))
E         )
E         
E         
E         CREATE TABLE registered_models (
E               name VARCHAR(256) NOT NULL,
E               creation_time BIGINT,
E               last_updated_time BIGINT,
E               description VARCHAR(5000),
E               CONSTRAINT registered_model_pk PRIMARY KEY (name),
E               UNIQUE (name)
E         )
E         
E         
E         CREATE TABLE experiment_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               experiment_id INTEGER NOT NULL,
E               CONSTRAINT experiment_tag_pk PRIMARY KEY (key, experiment_id),
E               FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id)
E         )
E         
E         
E         CREATE TABLE model_versions (
E               name VARCHAR(256) NOT NULL,
E               version INTEGER NOT NULL,
E               creation_time BIGINT,
E               description VARCHAR(5000),
E               user_id VARCHAR(256),
E               current_stage VARCHAR(20),
E               source VARCHAR(500),
E               run_id VARCHAR(32),
E               status VARCHAR(20),
E               status_message VARCHAR(500),
E               run_link VARCHAR(500),
E               CONSTRAINT model_version_pk PRIMARY KEY (name, version),
E               FOREIGN KEY(name) REFERENCES registered_models (name) ON UPDATE CASCADE
E         )
E         
E         
E         CREATE TABLE registered_model_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               name VARCHAR(256) NOT NULL,
E               CONSTRAINT registered_model_tag_pk PRIMARY KEY (key, name),
E               FOREIGN KEY(name) REFERENCES registered_models (name) ON UPDATE CASCADE
E         )
E         
E         
E         CREATE TABLE runs (
E               run_uuid VARCHAR(32) NOT NULL,
E               name VARCHAR(250),
E               source_type VARCHAR(20),
E               source_name VARCHAR(500),
E               entry_point_name VARCHAR(50),
E               user_id VARCHAR(256),
E               status VARCHAR(9),
E               start_time BIGINT,
E               end_time BIGINT,
E               source_version VARCHAR(50),
E               lifecycle_stage VARCHAR(20),
E               artifact_uri VARCHAR(200),
E               experiment_id INTEGER,
E               deleted_time BIGINT,
E               CONSTRAINT run_pk PRIMARY KEY (run_uuid),
E               FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id),
E               CONSTRAINT runs_lifecycle_stage CHECK (lifecycle_stage IN ('active', 'deleted')),
E               CONSTRAINT source_type CHECK (source_type IN ('NOTEBOOK', 'JOB', 'LOCAL', 'UNKNOWN', 'PROJECT')),
E               CHECK (status IN ('SCHEDULED', 'FAILED', 'FINISHED', 'RUNNING', 'KILLED'))
E         )
E         
E         
E         CREATE TABLE latest_metrics (
E               key VARCHAR(250) NOT NULL,
E               value FLOAT NOT NULL,
E               timestamp BIGINT,
E               step BIGINT NOT NULL,
E               is_nan BOOLEAN NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT latest_metric_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid),
E               CHECK (is_nan IN (0, 1))
E         )
E         
E         
E         CREATE TABLE metrics (
E               key VARCHAR(250) NOT NULL,
E               value FLOAT NOT NULL,
E               timestamp BIGINT NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               step BIGINT DEFAULT '0' NOT NULL,
E               is_nan BOOLEAN DEFAULT '0' NOT NULL,
E               CONSTRAINT metric_pk PRIMARY KEY (key, timestamp, step, run_uuid, value, is_nan),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid),
E               CHECK (is_nan IN (0, 1))
E         )
E         
E         
E         CREATE TABLE model_version_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               name VARCHAR(256) NOT NULL,
E               version INTEGER NOT NULL,
E               CONSTRAINT model_version_tag_pk PRIMARY KEY (key, name, version),
E               FOREIGN KEY(name, version) REFERENCES model_versions (name, version) ON UPDATE CASCADE
E         )
E         
E         
E         CREATE TABLE params (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(500) NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT param_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid)
E         )
E         
E         
E         CREATE TABLE tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT tag_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid)
E         )
E         
E         ===================================== DIFF =====================================
E           
E           CREATE TABLE alembic_version (
E               version_num VARCHAR(32) NOT NULL,
E               CONSTRAINT alembic_version_pkc PRIMARY KEY (version_num)
E           )
E           
E           
E           CREATE TABLE experiments (
E               experiment_id INTEGER NOT NULL,
E               name VARCHAR(256) NOT NULL,
E               artifact_location VARCHAR(256),
E               lifecycle_stage VARCHAR(32),
E         +     creation_time BIGINT,
E               last_update_time BIGINT,
E               CONSTRAINT experiment_pk PRIMARY KEY (experiment_id),
E               UNIQUE (name),
E               CONSTRAINT experiments_lifecycle_stage CHECK (lifecycle_stage IN ('active', 'deleted'))
E           )
E           
E           
E           CREATE TABLE registered_models (
E               name VARCHAR(256) NOT NULL,
E               creation_time BIGINT,
E               last_updated_time BIGINT,
E               description VARCHAR(5000),
E               CONSTRAINT registered_model_pk PRIMARY KEY (name),
E               UNIQUE (name)
E           )
E           
E           
E           CREATE TABLE experiment_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               experiment_id INTEGER NOT NULL,
E               CONSTRAINT experiment_tag_pk PRIMARY KEY (key, experiment_id),
E               FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id)
E           )
E           
E           
E           CREATE TABLE model_versions (
E               name VARCHAR(256) NOT NULL,
E               version INTEGER NOT NULL,
E               creation_time BIGINT,
E         +     last_updated_time BIGINT,
E               description VARCHAR(5000),
E               user_id VARCHAR(256),
E               current_stage VARCHAR(20),
E               source VARCHAR(500),
E               run_id VARCHAR(32),
E               status VARCHAR(20),
E               status_message VARCHAR(500),
E               run_link VARCHAR(500),
E               CONSTRAINT model_version_pk PRIMARY KEY (name, version),
E               FOREIGN KEY(name) REFERENCES registered_models (name) ON UPDATE CASCADE
E           )
E           
E           
E           CREATE TABLE registered_model_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               name VARCHAR(256) NOT NULL,
E               CONSTRAINT registered_model_tag_pk PRIMARY KEY (key, name),
E               FOREIGN KEY(name) REFERENCES registered_models (name) ON UPDATE CASCADE
E           )
E           
E           
E           CREATE TABLE runs (
E               run_uuid VARCHAR(32) NOT NULL,
E               name VARCHAR(250),
E               source_type VARCHAR(20),
E               source_name VARCHAR(500),
E               entry_point_name VARCHAR(50),
E               user_id VARCHAR(256),
E               status VARCHAR(9),
E               start_time BIGINT,
E               end_time BIGINT,
E               source_version VARCHAR(50),
E               lifecycle_stage VARCHAR(20),
E               artifact_uri VARCHAR(200),
E               experiment_id INTEGER,
E               deleted_time BIGINT,
E               CONSTRAINT run_pk PRIMARY KEY (run_uuid),
E               FOREIGN KEY(experiment_id) REFERENCES experiments (experiment_id),
E               CONSTRAINT runs_lifecycle_stage CHECK (lifecycle_stage IN ('active', 'deleted')),
E               CONSTRAINT source_type CHECK (source_type IN ('NOTEBOOK', 'JOB', 'LOCAL', 'UNKNOWN', 'PROJECT')),
E               CHECK (status IN ('SCHEDULED', 'FAILED', 'FINISHED', 'RUNNING', 'KILLED'))
E           )
E           
E           
E           CREATE TABLE latest_metrics (
E               key VARCHAR(250) NOT NULL,
E               value FLOAT NOT NULL,
E               timestamp BIGINT,
E               step BIGINT NOT NULL,
E               is_nan BOOLEAN NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT latest_metric_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid),
E               CHECK (is_nan IN (0, 1))
E           )
E           
E           
E           CREATE TABLE metrics (
E               key VARCHAR(250) NOT NULL,
E               value FLOAT NOT NULL,
E               timestamp BIGINT NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               step BIGINT DEFAULT '0' NOT NULL,
E               is_nan BOOLEAN DEFAULT '0' NOT NULL,
E               CONSTRAINT metric_pk PRIMARY KEY (key, timestamp, step, run_uuid, value, is_nan),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid),
E               CHECK (is_nan IN (0, 1))
E           )
E           
E           
E           CREATE TABLE model_version_tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               name VARCHAR(256) NOT NULL,
E               version INTEGER NOT NULL,
E               CONSTRAINT model_version_tag_pk PRIMARY KEY (key, name, version),
E               FOREIGN KEY(name, version) REFERENCES model_versions (name, version) ON UPDATE CASCADE
E           )
E           
E           
E           CREATE TABLE params (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(500) NOT NULL,
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT param_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid)
E           )
E           
E           
E           CREATE TABLE tags (
E               key VARCHAR(250) NOT NULL,
E               value VARCHAR(5000),
E               run_uuid VARCHAR(32) NOT NULL,
E               CONSTRAINT tag_pk PRIMARY KEY (key, run_uuid),
E               FOREIGN KEY(run_uuid) REFERENCES runs (run_uuid)
E           )
E         
E         ================================== HOW TO FIX ==================================
E         Manually copy & paste the expected schema in tests/db/schemas/sqlite.sql or run the following command:
E         docker-compose -f tests/db/docker-compose.yml run --rm mlflow-sqlite python tests/db/test_schema.py
```

Comment (harupy) [reply tests/db/test_schema.py:157]: This is a bit difficult to copy & paste though...

**Review by BenWilson2 — APPROVED** [review]
LGTM! Great addition :)

**Comment by BenWilson2** [comment]
@harupy The error message instruction for executing the docker command is outdated. I have an adjustment in this PR: https://github.com/mlflow/mlflow/pull/6937/files

**Pull request merged**